site/app/views/NavigationView.php

```diff
@@ -14,6 +14,7 @@ public function showGradeables($sections_to_list) {
         $semester = $this->core->getConfig()->getSemester();
         $course = $this->core->getConfig()->getCourse();
         $site_url = $this->core->getConfig()->getSiteUrl();
+        $columnSpan = 10;
```

Comment:
what is the point of this variable? I don't think it's referenced in the page anywhere.